test/testable.ml

```diff
@@ -0,0 +1,5 @@
+module Opam = struct
```

Comment:
Shouldn't this be defined in `test_opam_cmd.ml` ? I think `Opam_cmd` as a whole is not unit testable so this will probably never get shared between tests.

Reply:
The idea is that this type can be used elsewhere so I added a `Testable` module so different tests can reuse that but I agree it might be a little bit too early and since it's only used in a single place I'll move it back!